src/class/methods.rs

```diff
@@ -95,17 +95,21 @@ impl PyMethodDef {
     pub const fn cfunction_with_keywords(
         name: &'static str,
         cfunction: PyCFunctionWithKeywords,
-        flags: c_int,
```

Comment:
How about `Option<c_int>`?
Passing `None` would be (slightly) better than `0`.
I'm writing this because I don't like using `.flags()` pattern for a non-builder struct, but this is my personal taste, and please choose what you like.

Reply:
Thanks, I changed to do so.

Reply:
Ah, also with MSRV we cannot use `match` in `const fn`, so we'd need to store `Option<c_int>` in the struct and do slightly more work at runtime.

So I'm going to keep `.flags()` for now. Maybe in the future after we increase MSRV we can change to use `Option<c_int>`.